response_operations_ui/views/respondents.py

```diff
@@ -110,12 +154,106 @@ def respondent_details(respondent_id):
     elif info:
         flash(info, "information")
 
+    # Share Surveys and Pending Surveys information collection section
+    pending_surveys = party_controller.get_pending_surveys_by_party_id(respondent_id)
+    pending_transfer_surveys = []
+    pending_share_surveys = []
+    for pending_survey in pending_surveys:
+        if "is_transfer" in pending_survey and pending_survey["is_transfer"] is True:
+            pending_transfer_surveys.append(pending_survey)
+        else:
+            pending_share_surveys.append(pending_survey)
+    formatted_transfer_surveys = get_formatted_pending_surveys(pending_transfer_surveys)
+    formatted_share_surveys = get_formatted_pending_surveys(pending_share_surveys)
+
     return render_template(
         "respondent.html",
         respondent=respondent,
         enrolments=enrolments,
         breadcrumbs=breadcrumbs,
         mark_for_deletion=account["mark_for_deletion"],
+        pending_transfer_surveys=formatted_transfer_surveys,
+        pending_share_surveys=formatted_share_surveys,
+        respondent_id=respondent_id,
+    )
+
+
+def get_formatted_pending_surveys(pending_surveys: list):
```

Comment:
Minor comment, you can change this to
```python
def get_formatted_pending_surveys(pending_surveys: list) -> list:
```
and then you can remove the rtype and type from the docstring for this function

Reply:
ok